Fakebook.Posts/Fakebook.Posts.RestApi/Controllers/PostsController.cs

```diff
@@ -233,5 +239,27 @@ public async Task<IActionResult> GetNewsfeedAsync()
             return NoContent();
         }
 
+        [HttpPost("UploadPicture"), DisableRequestSizeLimit]
+        public async Task<ActionResult> UploadPicture() 
+        {
+            try {
+                if (Request.Form.Files[0] is IFormFile file)
+                {
+                    // generate a random guid from the file name
+                    var newFileName = $"{Request.Form["userId"]}-{Guid.NewGuid()}.{file.FileName.Split('.').Last()}";
```

Comment:
it'd be nice if an example file name were given to illustrate the format here. not a bad idea to factor that into a helper method either.

Reply:
Wrote a comment that shows what happens with a gif being uploaded